src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/PaneLayoutPreferencesPane.java

```diff
@@ -149,7 +149,7 @@ public void setSelectedIndices(ArrayList<Boolean> selected)
          ArrayList<String> value = new ArrayList<>();
          for (CheckBox checkBox : checkBoxes_)
          {
-            if (checkBox.getValue())
+            if (checkBox.getValue() && checkBox.getText() != "Presentation")
```

Comment:
Nit: in Java, comparison of strings with `==` and `!=` compares whether references are equal, so using `String.equals()` (or our `StringUtil.equals()`) is usually preferable.

In practice, because the code is compiled to JavaScript this will probably do the right thing, but IMHO it's best to "obey" the Java semantics here.

Reply:
Thanks.